src/client/datascience/jupyter/liveshare/hostJupyterServer.ts

```diff
@@ -337,6 +337,8 @@ export class HostJupyterServer extends LiveShareParticipantHost(JupyterServerBas
                 kernelInfo = kernelConnection;
             } else if (!launchInfo.connectionInfo.localLaunch && kernelConnection?.kind === 'connectToLiveKernel') {
                 kernelInfo = kernelConnection;
+            } else if (!launchInfo.connectionInfo.localLaunch && kernelConnection?.kind === 'startUsingKernelSpec') {
```

Comment:
Fix for remote native notebooks (we weren't starting the kernel spec selected in remote native)
New Tests picked this up.